### DISCUSSION TITLE
ZeroDivisionError

### DISCUSSION THREAD

**Discussion by Hydro2023** [jddingemanse/awtiCodeDev#25] [Q&A] [answered]
A ZeroDivisionError is reported by the Python interpreter when it detects a division operation is being performed and the denominator (bottom number) is 0. So Python treats this as an error condition or not ??

**Comment by jddingemanse** [comment, marked as answer]
A ZeroDivisionError is indeed one of the error types. For example, `10/0` will give a `ZeroDivisionError`. There are many different error types. `'hello'.captalze()` will give you for example an `AttributeError`, because `.captalze()` does not exist (it is a misspelling of `.capitalize()`.

Once you see one of those appearing on your screen, it means your code has stopped running. The error message with error type (e.g. ZeroDivisionError) tells you a bit about what kind of error it was, but it comes down to the same: Python encountered an error in your code and could therefore not continue. So, indeed, Python treats a ZeroDivisionError as an error.
